nautobot/core/fields.py

```diff
@@ -0,0 +1,10 @@
+from django_extensions.db.fields import AutoSlugField as _AutoSlugField
+
+
+class AutoSlugField(_AutoSlugField):
```

Comment:
It'd be good to have a docstring here, possibly taken more-or-less verbatim from https://django-extensions.readthedocs.io/en/latest/field_extensions.html#current-database-model-field-extensions?

Reply:
Sure thing, update coming.